generator/03-syndesis-ui.yml.mustache

```diff
@@ -83,14 +82,18 @@
           imagePullPolicy: IfNotPresent
 {{^Probeless}}
           livenessProbe:
-            httpGet:
-              path: "/"
-              port: 8080
+            exec:
+              command:
+              - curl
+              - -sSLf
```

Comment:
out of curiosity: Why using curl here and not relying on the integrated HTTP check ?

Reply:
Because we're now bound to localhost rather than an externally accessible port.